### DISCUSSION TITLE
sharp mx-3060N

### DISCUSSION THREAD

**Discussion by SimonGerm** [alexpevzner/sane-airscan#242] [Show and tell]
Hi,
thanks for your project, have at the moment
a sharp mx-3060n- multifunction printer 
attached to sane using your airscan:
must be enabled in printer:
system-settings/ network settings/ airprint settings/ airscan
have not tested every detail, but is scanning as expected
at an opensuse tumbleweed.
the offiicial sane-backends package did find
the device, after:
echo "escl" | tee /etc/sane.d/dll.d/escl 
but did not scan, "internal server error"
your package work.
airscan-discover: 
SHARP MX-3060N (6503134000) = .......

simoN

**Comment by alexpevzner** [comment]
Thank you for your feedback!

I've added your device to the list of supported devices.

If you wish, you may report `sane-escl` failure to the SANE mailing list: sane-devel@alioth-lists.debian.net or via their GitLab: https://gitlab.com/sane-project/backends/-/issues
